packages/opentelemetry-test-utils/package.json

```diff
@@ -38,7 +38,7 @@
     "@opentelemetry/api": "^1.0.2"
   },
   "devDependencies": {
-    "@opentelemetry/api": "1.0.2",
+    "@opentelemetry/api": "1.1.0",
```

Comment:
There are pinned versions in dependencies below.

Reply:
And these.